docs/index.rst

```diff
@@ -1,4 +1,4 @@
-MQTT state-machine protocol (mqttpproto)
+MQTT state-machine protocol (mqttproto)
 ========================================
```

Comment:
```suggestion
=======================================
```

Reply:
I would commit and merge but you opted not to allow maintainers to edit your PR.

Reply:
*Sigh* sorry about that. Fixed.